test/controllers/test_case_controller.py

```diff
@@ -21,6 +23,15 @@ def test_get_all_business_ids_for_collection_exercise(self, mock_engine):
         test_output = case_controller.get_all_business_ids_for_collection_exercise(EXERCISE_ID)
         self.assertEqual(expected_output, test_output)
 
+    @mock.patch("rm_reporting.app.party_db.begin")
+    def test_get_all_business_ids_for_collection_exercise_sqlalchemyerror(self, mock_error):
```

Comment:
Only thing is naming, you're not mocking the error but the db and then raising an error so mock_db seems like a more appropriate name

Reply:
Done.